__tests__/app.test.js

```diff
@@ -121,54 +121,42 @@ describe("Endpoint: GET /api/articles/:article_id", () => {
   });
 });
 
-// describe("Endpoint: GET /api/articles/:article_id/comments", () => {
-//   test("Returns a status code of 200 when query exists", () => {
-//     return request(app)
-//       .get("/api/articles/1/comments")
-//       .expect(200)
-//   });
-//   test("Returns an array of objects with the correct ids and properties", () => {
-//     return request(app)
-//       .get("/api/articles/1/comments")
-//       .expect(200)
-//       .then((result) => {
-//         console.log(result.body.comments);
-//         result.body.comments.forEach((comment) => {
-//           expect(comment).toMatchObject({
-//             comment_id: expect.any(Number),
-//             votes: expect.any(Number),
-//             created_at: expect.any(String),
-//             author: expect.any(String),
-//             body: expect.any(String),
-//             article_id: expect.any(Number)
-//           });
-//         });
-//       });
-//   });
-// });
+describe("Endpoint: GET /api/articles/:article_id/comments", () => {
+  test("Returns a status code of 200 when query exists", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+  });
+  test("Returns an array of objects with the correct ids and properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((result) => {
+        result.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number)
+          });
+        });
+      });
+  });
+});
```

Comment:
This also needs error handling, when passed an article_id which does not exist i.e, 999, or when passed an invalid id i.e. not-a-number